sdk/aqueduct/client.py

```diff
@@ -501,6 +508,7 @@ def publish_flow(
         k_latest_runs: Optional[int] = None,
         config: Optional[FlowConfig] = None,
         source_flow: Optional[Union[Flow, str, uuid.UUID]] = None,
+        use_local: Optional[bool] = False,
```

Comment:
You might need to rebase. `list_params()` is gone now.

Reply:
I will be doing the rebase in when `eng-2584` merge into `main` since I will be using from_local_data in that branch as well